client/router_create.go

```diff
@@ -50,6 +52,35 @@ func OauthProxyContainer(serviceAccount string, servicePort string) *corev1.Cont
 	}
 }
 
+func getExtraLabels() map[string]string {
+	var extraLabels = map[string]string{}
+
+	labels := DefaultSkupperExtraLabels
+	envLabels, envFound := os.LookupEnv("SKUPPER_EXTRA_LABELS")
+
+	if envFound {
+		if labels == "" {
+			labels = envLabels
+		} else {
+			labels = labels + "," + envLabels
+		}
+	}
+
+	if labels != "" {
+		// "key1:value1,key2:value2" where key and value are lowercase RFC 1123 compliant
+		labelRegex := regexp.MustCompile(`[a-z0-9]([-a-z0-9]*[a-z0-9]):[a-z0-9]([-a-z0-9]*[a-z0-9])+(,[a-z0-9]([-a-z0-9]*[a-z0-9]):[a-z0-9]([-a-z0-9]*[a-z0-9])+)*`)
```

Comment:
This regexp could be defined as a public variable as I see potential for it being
reused in other places.